proposals/0293-extend-property-wrappers-to-function-and-closure-parameters.md

```diff
@@ -614,27 +701,24 @@ Otherwise:
 ```swift
 @propertyWrapper
 struct Reference<Value> {
-    
   var wrappedValue: Value {
-    get 
-    nonmutating set
+    get { ... }
+    nonmutating set { ... }
   }
-    
   var projectedValue: Reference<Value> {
     self
   }
-  
 }
 
-func useReference(@Reference @Asserted(.nonEmpty) reference: String) {
+func useReference(@Reference @Logged reference: String) {
```

Comment:
Before the code examples it says:

> **Example**: Consider the following `Reference` property wrapper, which is composed with `Asserted` and used on a function parameter:

When it should be:


> **Example**: Consider the following `Reference` property wrapper, which is composed with `Logged` and used on a function parameter:


In the conclusion it says:

> Since both the getter and setter of `Reference.wrappedValue` are `nonmutating`, a setter can be synthesized for `var reference`, even though `Asserted.wrappedValue` [...]

When it should be:


> Since both the getter and setter of `Reference.wrappedValue` are `nonmutating`, a setter can be synthesized for `var reference`, even though `Logged.wrappedValue` [...]